dist/sonarqube-scanner-params.js

```diff
@@ -9,21 +9,26 @@ var uniq = require('lodash.uniq')
 
 module.exports = defineSonarQubeScannerParams
 
+var invalidCharacterRegex = /[?$*+~.()'"!:@]/g
```

Comment:
I think the forward slash "/" needs to be removed as well. I checked this out and ran a docker version of sonarqube 7.1 ([source](https://hub.docker.com/_/sonarqube/)) and tried a scoped module I have and got the following error where I have crossed out any identifiable information (module name changed to "my/module", sorry it's a closed source repo).

```
INFO: Process project properties
INFO: ------------------------------------------------------------------------
INFO: EXECUTION FAILURE
INFO: ------------------------------------------------------------------------
INFO: Total time: 1.121s
INFO: Final Memory: 9M/279M
INFO: ------------------------------------------------------------------------
ERROR: Error during SonarQube Scanner execution
ERROR: Validation of project reactor failed:
  o "my/module" is not a valid project or module key. Allowed characters are alphanumeric, '-', '_', '.' and ':', with at least one non-digit.
ERROR: 
```

I had started on an implementation of this and opened PR #53 which also includes a unit test to check this for a basic scoped package name which might be of interest to you.

Reply:
I've created a PR into your branch here: https://github.com/raksit31667/sonar-scanner-npm/pull/1
It removes the forward slash and adds the unit tests. I will close down PR #53 in favor of this one with the added changes.

Reply:
@raksit31667 Good point from @Bangsadrengur, I wait that you apply his PR on yours before proceeding with the review and merge of the current PR.

Reply:
@bellingard Fixed, please merge this :white_check_mark: